packages/block-editor/src/components/date-format-control/README.md

```diff
@@ -0,0 +1,64 @@
+# DateFormatControl
+
+The `DateFormatControl` component renders a dropdown that lets the user choose a
+_date format_. That is, how they want their dates to be formatted.
+
+A user can either pick _Site default_, a suggested format, or _Custom_.
+
+_Site default_ is equivalent to no selection. It means that the user wants their
+dates to be formatted however their website is configured to format dates.
+
+Selecting _Custom_ will display a text box that lets the user type in any format
+string.
+
+All date format strings should be in the format accepted by by the (`dateI18n`
+function in
+`@wordpress/date`)[https://github.com/WordPress/gutenberg/tree/trunk/packages/date#datei18n].
```

Comment:
Tiny formatting issue: looks like this markdown syntax is the wrong way around 😀 (should be `[]` followed by `()`)

![image](https://user-images.githubusercontent.com/14988353/157353756-0dcbacbd-8451-44b9-853f-9824c3a1809c.png)

Reply:
Gah, every time 😂